configure

```diff
@@ -375,6 +379,31 @@ parser.add_option('--enable-static',
     dest='enable_static',
     help='build as static library')
 
+parser.add_option('--static-libstdc++',
+    action='store_true',
+    dest='use_static_libstdcpp',
+    help='statically link to libstdc++ library instead of dynamically linking')
```

Comment:
Not sure, although there is another reference to -static-libstdc++ in the configure script. I've removed it.